src/components/select_gateway_node.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <v-toolbar-title>Select Gateway Node</v-toolbar-title>
+  <v-autocomplete
+    label="Choose your node"
+    :items="items"
+    :loading="loading"
+    item-title="publicConfig.domain"
+    item-value="nodeId"
+    v-model="selectedNode"
+  >
+    <template v-slot:append-item v-if="!noMoreResults">
+      <div v-intersect="handleLoadMoreGateWayNodes" class="pa-4 teal--text">
+        Loading more items ...
+      </div>
+    </template>
+  </v-autocomplete>
+</template>
+
+<script lang="ts" setup>
+import { onMounted, ref, watch } from 'vue'
+import { loadGateWayNodes } from '../utils/gateway'
+import { useProfileManager } from '../stores'
+import { getGrid } from '../utils/grid'
+import type { GridClient, NodeInfo } from 'grid3_client'
+
+//states
+defineProps<{ modelValue: number }>()
```

Comment:
Make modelValue optional so user doesn't have to specify initial value
```suggestion
defineProps<{ modelValue?: number }>()
```